schematic/schemas/explorer.py

```diff
@@ -522,7 +526,7 @@ def update_class(self, class_info):
         validate_class_schema(class_info)
         self.schema["@graph"].append(class_info)
         validate_schema(self.schema)
-        print("Updated the class {} successfully!".format(class_info["rdfs:label"]))
+        logger.info(f"""Updated the class {class_info["rdfs:label"]} successfully.""")
```

Comment:
As a heads-up, you can use different quotes to avoid the triple-quote:

```
logger.info(f"Updated the class {class_info['rdfs:label']} successfully.")
```

Reply:
Thanks @BrunoGrandePhD, I've made the f-string formatting change everywhere.